django_project/trail/models/trail.py

```diff
@@ -0,0 +1,92 @@
+# coding=utf-8
+"""Model definitions for a trail app"""
+
+import os
+import uuid as uuid_lib
+
+from django.contrib.gis.db import models
+from django.utils.translation import ugettext_lazy as _
+from django.utils.text import slugify
+from django.conf.global_settings import MEDIA_ROOT
+
+from unidecode import unidecode
+
+from core.settings.contrib import STOP_WORDS
+# from ..utils import GUIDModel
+
+__author__ = 'Alison Mukoma <alison@kartoza.com>'
+__date__ = '01/09/2018'
+__license__ = "GPL"
+__copyright__ = 'kartoza.com'
+
+
+class Trail(models.Model):
+    "Model definition of a Trail."
```

Comment:
" -> """

Is the code not failing with those single qotes? Other docstrings should also be placed in """ triple quotes.